Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -72,7 +72,14 @@ void MemoryViewWidget::Update()
   if (rowCount() == 0)
     setRowCount(1);
 
-  setRowHeight(0, 24);
+  const QFontMetrics fm(Settings::Instance().GetDebugFont());
```

Comment:
IIRC, getting the font metrics is pretty expensive, to the point that it was noticeable to do it every update in some other widgets. You should probably store the data you need in member variables and register the `Settings::DebugFontChanged` event.

Reply:
Alright, thanks!